_includes/scripts.html

```diff
@@ -4,6 +4,9 @@
   {% for js_file in page.custom_js %}
   <script src='/assets/js/{{ js_file }}.js' type="text/javascript"></script>
   {% endfor %}
+{% elsif layout.custom_js %}
+  <script src='/assets/js/guides.js' type="text/javascript"></script>
+  <script src='/assets/js/prism.js' type="text/javascript"></script>
```

Comment:
If you're going to go with the `layout.custom_js` approach, I think you'd want to loop over the contents of `custom_js` as in `page.custom_js` above, rather than hard-coding these two.